test/javascripts/components/load-more-test.js.es6

```diff
@@ -0,0 +1,28 @@
+import { _setTestParams } from "discourse/lib/eyeline";
+import componentTest from "helpers/component-test";
+
+moduleForComponent("load-more", { integration: true });
+
+componentTest("preview updates with markdown", {
+  template: `
+    {{#load-more selector=".numbers tr" action=(action loadMore)}}
+      <table class='numbers'>
+        <tr><td>1</td></tr>
+        <tr><td>2</td></tr>
+        <tr><td>3</td></tr>
+      </table>
+    {{/load-more}}`,
+
+  beforeEach() {
+    this.set("loadMore", () => this.set("loadedMore", true));
+    _setTestParams(false, true);
+  },
+
+  afterEach() {
+    _setTestParams(true);
```

Comment:
might be worth to have a:  `defaultEyeLineBehavior` function to call, so we don't have to worry about default eyeline behavior changing in the future, which could lead to hard to track side effects.

Reply:
Calling `configureEyeline` with no parameter will reset its behaviour.